stdlib/public/core/Hasher.swift

```diff
@@ -331,16 +360,22 @@ public struct _Hasher {
     _core.combine(value)
   }
 
+  //FIXME: Convert to @usableFromInline internal once integers hash correctly.
   @effects(releasenone)
   public mutating func _combine(bytes value: UInt64, count: Int) {
     _core.combine(bytes: value, count: count)
   }
 
+  /// Feed the contents of `buffer` into this hasher, mixing it into the hasher
+  /// state.
   @effects(releasenone)
   public mutating func combine(bytes: UnsafeRawBufferPointer) {
     _core.combine(bytes: bytes)
   }
 
+  /// Finalize the hasher state and return the hash value.
+  /// Finalizing invalidates the hasher; additional bits cannot be combined
+  /// into it, and it cannot be finalized again.
```

Comment:
You're right, it isn't! I have to change it to be nonmutating but consuming. I don't think we need to change the name to `finalized()` because of the consuming semantics. @jckarter, do you agree?

Reply:
We didn't discuss changing the name. The discussion about the hasher being invalidated may still not be necessary anymore since the consuming means you can't use the finalized value anymore anyway.